Makefile

```diff
@@ -40,6 +40,7 @@ all: ocs-operator ocs-must-gather ocs-registry
 	cluster-clean \
 	ocs-operator-openshift-ci-build \
 	functest \
+        shellcheck-test \
```

Comment:
Now the splitting of the commits in the makefile is slightly off: the commit "Add shellcheck test target" only moves the `.PHONY` entry from a separate entry above the "shellcheck-test" target (which is legitmate), to the global `.PHONY` list (which is possibly more aligned with the current style of the Makefile.

It does not make a lot of sense to add a phony entry in one place and one commit and then move it to another place in a later commit. I assume you meant to have the addition of the shellcheck-test target in this commit as well?